devolutions-gateway/src/api/heartbeat.rs

```diff
@@ -6,6 +6,7 @@ use crate::extract::HeartbeatReadScope;
 use crate::http::HttpError;
 use crate::DgwState;
 use devolutions_agent_shared::windows::registry::get_installed_product_version;
+#[cfg(windows)]
 use devolutions_agent_shared::windows::AGENT_UPDATE_CODE;
```

Comment:
Instead of putting many `#[cfg(windows)]` inside the heartbeat module of the Gateway, could use expose a cross-platform API from `devolutions_agent_shared`, returning `None` on all platforms except Windows (for now)?

Reply:
Good idea, I moved cfg to `devolutions_agent_shared`